kii-ambassadors.md

```
@@ -1,117 +1,139 @@
 ---
-description: Become a Kii Ambassador
+description:  Become a Kii Ambassador — Connect, Grow, and Earn with KiiChain 
 ---
 
 # Kii Ambassadors
 
-### What does it mean to become a KiiAmbassador?&#x20;
+### What does it mean to become a Kii Ambassador?
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)